src/timeunit.ts

```diff
@@ -294,7 +294,8 @@ export function template(timeUnit: TimeUnit, field: string, shortTimeLabels: boo
    const template = '{{' + field + ' | time:\'' + out.join(' ') + '\'}}';
 
    // FIXME: Remove this RegExp Hack!!!
-   return template.replace(new RegExp('{{' + field + ' \\| time:\'\'}}', 'g'), '');
+   const escapedField = field.replace(/([.*+?^=!:${}()|\[\]\/\\])/g, '\\$1');
```

Comment:
Just addressed this in last comment :) we're on the same wavelength. This is necessary because 'field' now has square brackets which need to be escaped in Regex (the dot notation did not need escaping).

I'll open an issue to rewrite this. Didn't feel like it made sense within this PR